intellij-haxe.iml

```diff
@@ -13,9 +13,10 @@
       <sourceFolder url="file://$MODULE_DIR$/src/icons" isTestSource="false" />
```

Comment:
Do not check this file in.  We want to leave the .iml in the "configured for 14.1" state.